SwiftUIBasics/Views/RatingView.swift

```diff
@@ -7,12 +7,23 @@
 
 import SwiftUI
 
-struct RatingView: View {
+
+struct StartRaiting : View {
+    @State private var raiting : Int = 0
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        HStack{
+            ForEach(1..<6){ index in
+                Image(systemName: index <= raiting ? "star.fill" : "star" )
+                    .foregroundStyle(.yellow)
+                .onTapGesture {
+                    raiting = index
+                }
```

Comment:
what will happened if I want to remove my review by tapping in a star that is already filled?